opentelemetry-zipkin/src/exporter/env.rs

```diff
@@ -0,0 +1,66 @@
+use std::env;
+use std::time::Duration;
+use crate::ZipkinPipelineBuilder;
+
+/// HTTP endpoint for Zipkin collector.
+/// e.g. "http://localhost:9411/api/v2/spans"
+#[cfg(any(
+    feature = "reqwest-client",
+    feature = "surf-client",
+    feature = "reqwest-blocking-client"
+))]
+const ENV_ENDPOINT: &str = "OTEL_EXPORTER_ZIPKIN_ENDPOINT";
+
+/// Maximum time the Zipkin exporter will wait for each batch export
+#[cfg(any(
+    feature = "reqwest-client",
+    feature = "surf-client",
+    feature = "reqwest-blocking-client"
+))]
+const ENV_TIMEOUT: &str = "OTEL_EXPORTER_ZIPKIN_TIMEOUT";
+
+/// Assign builder attributes from env
+pub(crate) fn assign_attrs(mut builder: ZipkinPipelineBuilder) -> ZipkinPipelineBuilder {
+
+    #[cfg(any(
+        feature = "reqwest-client",
+        feature = "surf-client",
+        feature = "reqwest-blocking-client"
+    ))]
+    {
+        if let Some(endpoint) = env::var(ENV_ENDPOINT).ok().filter(|var| !var.is_empty()) {
+            builder = builder.with_collector_endpoint(endpoint);
+        }
+    }
+
+    #[cfg(any(
+        feature = "reqwest-client",
+        feature = "surf-client",
+        feature = "reqwest-blocking-client"
+    ))]
+    {
+        if let Some(timeout) = env::var(ENV_TIMEOUT).ok().filter(|var| !var.is_empty()) {
+            let timeout = Duration::from_millis(timeout.parse().unwrap_or_else(|e| panic!("{} Misformatted: {}",ENV_TIMEOUT,e)));
+
+            #[cfg(feature = "reqwest-blocking-client")]
+            let client = reqwest::blocking::Client::builder().timeout(timeout).build().unwrap();
+            #[cfg(all(
+                not(feature = "reqwest-blocking-client"),
+                not(feature = "surf-client"),
+                feature = "reqwest-client"
+            ))]
+            let client = reqwest::Client::builder().timeout(timeout).build().unwrap();
+            #[cfg(all(
+                not(feature = "reqwest-client"),
+                not(feature = "reqwest-blocking-client"),
+                feature = "surf-client"
+            ))]
+            let client: surf::Client = surf::Config::new()
+                .set_timeout(Some(timeout)).into();
```

Comment:
We may want to move the client initialization into this function instead of `default` and we should probably provide a default value of `10s` if the env var is not set. 

I see some of the client builders return `Result`. Generally, we want to avoid `unwrap`. Maybe we can fall back to the client without a timeout in this case. Although I don't expect it will happen